livestyled/schemas/merchant_account.py

```diff
@@ -0,0 +1,21 @@
+from marshmallow import EXCLUDE, fields, Schema
```

Comment:
I think we should have a `schemas/payment.py` module and then have all payment related schemas in one place. 

This will allow us to have this `livestyled.schemas.payment import MerchantSchema, PaymentGateway` an example is the `schema/league_table.py`. We should group related schemas in the same module in this case Merchant and Payment gateway.